modules/search/backend_search_block.go

```diff
@@ -0,0 +1,187 @@
+package search
```

Comment:
i would like to find a better place to put this code. all other modules are controlled through modules.go and are managed via the Cortex service manager.

Reply:
I guess we need to determine if this belongs in tempodb or not.  If not, /pkg/search might be ok.

Reply:
Discussed offline and decided to move it to /tempodb/search.